coldfront/core/allocation/urls.py

```diff
@@ -8,6 +8,8 @@
          allocation_views.AllocationCreateView.as_view(), name='allocation-create'),
     path('<int:pk>/', allocation_views.AllocationDetailView.as_view(),
          name='allocation-detail'),
+     path('<int:pk>/review-eula', allocation_views.AllocationEULAView.as_view(),
```

Comment:
This URL should only be added if `ALLOCATION_EULA_ENABLE` is True.

Reply:
Done